src/Sulu/Bundle/MediaBundle/Entity/MediaDataProviderRepository.php

```diff
@@ -238,7 +238,7 @@ protected function appendSortByJoins(QueryBuilder $queryBuilder, $alias, $locale
             ->setParameter('locale', $locale);
     }
 
-    public function createQueryBuilder($alias, $indexBy = null)
+    public function createQueryBuilder($alias, $indexBy = null): \Doctrine\ORM\QueryBuilder
```

Comment:
Return type should not be necessary here, because of the phpdoc and this will be a BC break